Clicker/ViewControllers/CardController+Extension.swift

```diff
@@ -15,14 +15,17 @@ extension CardController: ListAdapterDataSource {
         switch state {
         case .horizontal:
             if (currentIndex > -1) {
+                collectionView.isScrollEnabled = true
                 return pollsDateArray[currentIndex].polls
             } else {
+                collectionView.isScrollEnabled = false
                 return [EmptyStateModel(userRole: userRole)]
             }
         default:
-            return pollsDateArray.compactMap({ pollsDateModel -> PollDateModel? in
+            return pollsDateArray.enumerated().compactMap({ (i,pollsDateModel) -> PollDateModel? in
```

Comment:
rename i to index and you can do `pollsDateArray.enumerated().compactMap { ... }` no need for extra set of `()`